controller/golden_files_test.go

```diff
@@ -81,6 +87,9 @@ func testableCompareWithGolden(update bool, goldenFile string, actualObj interfa
 	if err != nil {
 		return errs.WithStack(err)
 	}
+	if len(stripDownToPattern) > 1 {
```

Comment:
@kwk Why are accepting []strings (`stripDownToPattern ...string`) in function and then checking it's length to be 1? If this is the case, shouldn't the `stripDownToPattern` be a `string` and not `...string`?

Reply:
So why not change `...string` to just `string` ? We can later update this to support multiple patterns